spec/features/updating_event_spec.rb

```diff
@@ -107,6 +122,7 @@
     fill_in "Enter staff email", with: email
     click_button "add_staff"
     click_link "Preview"
+    sleep 2
```

Comment:
Avoid manipulating test with `sleep`. Let the test behave exactly like the user is meant to. or... is the User meant to sleep on the page? 😀